tests/bootstrap.php

```diff
@@ -2,9 +2,13 @@
 defined('YII_DEBUG') || define('YII_DEBUG', true);
 defined('YII_ENV') || define('YII_ENV', 'test');
 
-Yii::$container = new \yii\di\Container();
+call_user_func(function() {
```

Comment:
What is the reason for adding the closure here? I see no scope to guard.

Reply:
I copied the bootstrap file inside yii advanced template (was getting errors that Yii was undefined)

Reply:
I have reverted back. The reason I was getting errors was because vendor/autoload.php was not loaded so I am including this also. Latest push essentially just removes the closure as you rightly noticed.